fe/src/main/java/org/apache/doris/load/loadv2/LoadingTaskPlanner.java

```diff
@@ -201,4 +200,17 @@ private String convertBrokerDescPartitionInfo() throws LoadException, MetaNotFou
         }
         return result;
     }
+
+    // when retry load by reusing this plan in load process, the load_id should be changed
+    public void updateLoadId(TUniqueId loadId) {
+        for (PlanFragment planFragment : fragments) {
+            if (!(planFragment.getSink() instanceof OlapTableSink)) {
+                continue;
+            }
+            OlapTableSink olapTableSink = (OlapTableSink) planFragment.getSink();
+            olapTableSink.updateLoadId(loadId);
+        }
+
+        LOG.info("update olap table sink's load id to {}, job: {}", DebugUtil.printId(loadId), loadJobId);
```

Comment:
Delete this debug log

Reply:
Link loadId and jobId is necessary.

Reply:
> Delete this debug log

No, this is not a debug log. We can not find the new load id without this bug